### PR TITLE
Handle unreachable network in HTTP fallback logic (SOFTWARE-3034)

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

**Comment by matyasselmeci** [comment]
better yet, print a warning inside the loop in case the error is different for HTTPS vs HTTP

File: osgpkitools/ConnectAPI.py [hunk 1]
```diff
@@ -59,14 +59,15 @@ def request_unauthenticated(self, config, name, email, phone, csr, vo='', cc_lis
         for conn in [httplib.HTTPSConnection(host_no_port), httplib.HTTPConnection(host_no_port)]:
```
Comment (matyasselmeci) [inline osgpkitools/ConnectAPI.py]: Need to add `last_network_error = None` right above this to avoid potential NameError below

**Comment by brianhlin** [comment]
I'm not sure that I want to show any HTTPS connection errors: that could be confusing for the user that's not aware of the HTTP fallback logic

**Comment by matyasselmeci** [comment]
That's fine. If you merge the potential NameError I pointed out, I'll merge this.

**Comment by brianhlin** [comment]
Done

**Pull request merged**